### ISSUE TITLE
Error when explicitly enabling Naming/BlockForwarding

### ISSUE THREAD

**Issue by JoeCohen** [rubocop/rubocop#10821]
Rubocop warns me to explicitly enable/disable `Naming/BlockForwarding`, 
but gives this error when the cop is explicitly enabled:
`Error: unrecognized cop or department Enabled found in .rubocop.yml`

NOTE: I'm on ruby 3.0.4, but the cop applies only to ruby >=3.1

--------

## Expected behavior

rubocop runs without the `Error: unrecognized cop or department Enabled found in .rubocop.yml` error
and/or
rubocop does not warn me to explicitly enable/disable `Naming/BlockForwarding`

## Actual behavior

$ rubocop -f html -o rubocop.html --ignore-parent-exclusion --only Naming --debug
For /vagrant/mushroom-observer: configuration from /vagrant/mushroom-observer/.rubocop.yml
configuration from /home/vagrant/.rvm/gems/ruby-3.0.4/gems/rubocop-performance-1.14.0/config/default.yml
configuration from /home/vagrant/.rvm/gems/ruby-3.0.4/gems/rubocop-performance-1.14.0/config/default.yml
Default configuration from /home/vagrant/.rvm/gems/ruby-3.0.4/gems/rubocop-1.31.1/config/default.yml
configuration from /home/vagrant/.rvm/gems/ruby-3.0.4/gems/rubocop-rails-2.14.2/config/default.yml
configuration from /home/vagrant/.rvm/gems/ruby-3.0.4/gems/rubocop-rails-2.14.2/config/default.yml
Inheriting configuration from /vagrant/mushroom-observer/.rubocop_todo.yml
.rubocop.yml: Metrics/AbcSize:Max overrides the same parameter in .rubocop_todo.yml
.rubocop.yml: Metrics/ClassLength:Max overrides the same parameter in .rubocop_todo.yml
.rubocop.yml: Metrics/MethodLength:Max overrides the same parameter in .rubocop_todo.yml
.rubocop.yml: Metrics/ModuleLength:Max overrides the same parameter in .rubocop_todo.yml
Error: unrecognized cop or department Enabled found in .rubocop.yml
Finished in 0.35871090900036506 seconds

## Steps to reproduce the problem

When running Rubocop, without explicitly enabling Naming/BlockForwarding, I get a message that includes:
>The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file. 
...
Naming/BlockForwarding: # new in 1.24 
Enabled: true 
...

So far, so good. 
But I when I explicitly enable the cop, with the following in `.rubocop.yml`
```yml
Naming/BlockForwarding: # new in 1.24
Enabled: true
```
I get an error when running rubocop:
```
$ rubocop -f html -o rubocop.html --ignore-parent-exclusion --only Naming
Error: unrecognized cop or department Enabled found in .rubocop.yml
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ bundle exec rubocop -V
1.31.1 (using Parser 3.1.2.0, rubocop-ast 1.18.0, running on ruby 3.0.4 x86_64-linux)
  - rubocop-performance 1.14.0
  - rubocop-rails 2.14.2

```

**Comment by dvandersluis** [comment]
You have to indent `Enabled` one level deeper than `Naming/BlockForwarding`.

**Comment by JoeCohen** [comment]
Thanks for the fast response and and my apologies!
(I could have sworn that I did indent it. But you are absolutely correct; I failed to do so.)
